Ada Examples/Examples Based On OpenGL Superbible/Simplest/just_a_dot.gpr

```diff
@@ -0,0 +1,194 @@
+limited with "../../../Ada_Source/OpenGLAda-master/opengl-glfw.gpr";
```

Comment:
This is not a portable path, please change it to just `../../../opengl-glfw.gpr`.